src/ucp/wireup/ep_match.c

```diff
@@ -110,13 +116,17 @@ ucp_ep_h ucp_ep_match_retrieve(ucp_worker_h worker, uint64_t dest_uuid,
 
 void ucp_ep_match_remove_ep(ucp_worker_h worker, ucp_ep_h ep)
 {
+    ucs_conn_match_queue_type_t conn_queue_type;
+
     if (!(ep->flags & UCP_EP_FLAG_ON_MATCH_CTX)) {
         return;
     }
 
+    conn_queue_type = (ep->flags & UCP_EP_FLAG_REMOTE_ID) ?
+                      UCS_CONN_MATCH_QUEUE_UNEXP : UCS_CONN_MATCH_QUEUE_EXP;
     ucs_conn_match_remove_elem(&worker->conn_match_ctx,
                                &ucp_ep_ext_gen(ep)->ep_match.conn_match,
-                               !(ep->flags & UCP_EP_FLAG_REMOTE_ID));
+                               conn_queue_type);
```

Comment:
can avoid `conn_queue_type` variable and pass the result of ternary operator directly

Reply:
ok, done